### DISCUSSION TITLE
JOTX-DTV

### DISCUSSION THREAD

**Discussion by solo489** [iptv-org/community#940] [Show and tell]
### Channel ID

JOTXDTV.jp

### Stream URL (optional)

https://redlabmcdn.s.llnwi.net/nv02/ryowa6hd/manifest.mpd

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
On both endpoints none of those works. The MPD output has been disabled by the owners of that server.

**Comment by solo489** [comment]
Replace the old link with a new one

**Comment by freearhey** [comment]
https://redlabmcdn.s.llnwi.net/nv02/ryowa6hd/manifest.mpd (offline)

![image](https://github.com/user-attachments/assets/166a2eaf-7684-47f7-bc04-59646ff31c15)
